docs/docs/07-forms.ru-RU.md

```diff
@@ -0,0 +1,175 @@
+
+```javascript
+  getInitialState: function() {
+    return {value: 'Hello!'};
+  },
+  handleChange: function(event) {
+    this.setState({value: event.target.value});
+  },
+  render: function() {
+    return (
+      <input
+        type="text"
+        value={this.state.value}
+        onChange={this.handleChange}
+      />
+    );
+  }
+```
+
+В этом примере мы принимаем значение, переданное пользователем, и обновляем свойство `value` компонента `<input>`. С помощью этого шаблона легко реализовывать интерфейсы, которые проверяют или отвечают на пользовательские взаимодействия. Например:
+
+```javascript
+  handleChange: function(event) {
+    this.setState({value: event.target.value.substr(0, 140)});
+  }
+```
+
+Этот метод принимает входные данные от пользователя и усекает значение до первых 140 символов.
+
+**Управляемый** компонент не поддерживает свое собственное внутреннее состояние; такой компонент отрисовывается исключительно
+на основании свойств.
+
+### Потенциальные проблемы с флажками и переключателями
+
+Примите во внимание, что в попытке нормализовать обработку изменений флажков и переключателей React использует событие `click` вместо события `change`. По большей части это приводит к ожидаемому поведению, за исключением вызова `preventDefault` в обработчике `change`. `preventDefault` предотвращает визуальное обновление поля ввода браузером, даже если свойство `checked` переключается на противоположное. Обойти это можно, удалив вызов `preventDefault` или поместив переключение `checked` в `setTimeout`.
+
+## Неуправляемые компоненты
+
+`<input>` без свойства `value` является *неуправляемым* компонентом:
+
+```javascript
+  render: function() {
+    return <input type="text" />;
+  }
+```
+
+Так будет отрисовано поле ввода, изначально имеющее пустое значение. Любой ввод от пользователя будет немедленно отражен в отрисованном элементе. Если вы хотите следить за обновлениями значения, вы можете использовать событие `onChange` так же, как с управляемыми компонентами.
+
+**Неуправляемый** компонент сам поддерживает свое собственное внутреннее состояние.
+
+### Значение по умолчанию
+
+Если вы хотите проинициализировать компонент непустым значением, вы можете передать свойство `defaultValue`. Например:
+
+```javascript
+  render: function() {
+    return <input type="text" defaultValue="Hello!" />;
+  }
+```
+
+Этот пример будет работать подобно примеру с **управляемыми компонентами** выше.
+
+Аналогично, `<input type="checkbox">` и `<input type="radio">` поддерживают `defaultChecked`, а `<select>` поддерживает `defaultValue`.
+
+> Замечание:
+>
+> Свойства `defaultValue` и `defaultChecked` используются только во время начальной отрисовки. Если вам нужно обновить значение при последующей отрисовке, вам необходимо использовать управляемый компонент.
```

Comment:
On this line I had to remove a link to another section altogether due to the issue described in https://github.com/facebook/react/pull/7235#discussion-diff-70165133R52. In this document, multiple section links are generated with the anchor `#-`, so it's impossible to refer to any one of them specifically.